src/generator/AutoRest.Go/Model/MethodGo.cs

```diff
@@ -213,6 +214,10 @@ public string MethodReturnSignature(bool helper)
             return $"{retVal}, {errVal}";
         }
 
+        public bool ShouldValidate() {
+            return !ParameterValidations.IsNullOrEmpty();
```

Comment:
I like the changes in ParameterGo.cs. Why gen code that wont be written to the file anyways. Still will change code here.

---
In reply to: [130652897](https://github.com/Azure/autorest/pull/2490#discussion_r130652897) [](ancestors = 130652897)